libphal/phal_sbe.C

```diff
@@ -326,35 +341,42 @@ void getDump(struct pdbg_target *chip, const uint8_t type, const uint8_t clock,
 	log(level::INFO, "Enter: getDump(%d) on %s", type,
 	    pdbg_target_path(chip));
 
-	if (is_ody_ocmb_chip(chip))
-	{
-		auto ret = sbe_dump(chip, type, clock, faCollect, data, dataLen);
-		if (ret != 0)
-		{
-			log(level::ERROR, "getDump(%s) failed", pdbg_target_path(chip));
+	if (is_ody_ocmb_chip(chip)) {
+		// sbe_ffdc_get internall uses chipop target so probe it
+		pdbg_target *co = get_ody_chipop_target(chip);
+		if (pdbg_target_probe(co) != PDBG_TARGET_ENABLED) {
+			log(level::ERROR, "failed to get ody chipop target",
+			    pdbg_target_path(chip));
+			return;
 		}
-		return;
-	}
-
-	// Validate input target is processor target.
-	validateProcTgt(chip);
+		auto ret =
+		    sbe_dump(chip, type, clock, faCollect, data, dataLen);
+		if (ret != 0) {
+			log(level::ERROR, "getDump(%s) failed",
+			    pdbg_target_path(chip));
+			throw captureFFDC(chip);
+		}
+	} else { // proc
+		// Validate input target is processor target.
+		validateProcTgt(chip);
 
-	if (!isTgtPresent(chip)) {
-		log(level::ERROR, "getDump(%s) Target is not present",
-		    pdbg_target_path(chip));
-	}
-	// SBE halt state need recovery before dump chip-ops
-	sbeHaltStateRecovery(chip);
+		if (!isTgtPresent(chip)) {
```

Comment:
From LN363 to LN374 I guess there is some tab/space issues. Kindly have a look.

Reply:
earlier spaces were used fixed the same to use tabs, as in IPL we use tabs